### PR TITLE
added new logo

### PR DESCRIPTION
**Pull request by BMG93**
replaced logo as per 
https://backstage.io/docs/getting-started/app-custom-theme/#overriding-backstage-and-material-ui-components-styles

### CODE REVIEW

File: packages/app/src/components/Root/LogoFull.tsx [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
-    width: 'auto',
-    height: 30,
-  },
-  path: {
-    fill: '#7df3e1',
-  },
-});
+import MyCustomLogoFull from './logo/Tux.svg.png'
 const LogoFull = () => {
-  const classes = useStyles();
```
Comment (MennaTullahTaha) [inline packages/app/src/components/Root/LogoFull.tsx:14]: When pushing code to main, make sure to delete all the commented code to keep the code clean.

**Review by MennaTullahTaha — COMMENTED** [review]
@DrAdamRoe I don't know if this logo should be merged to main. I ran the code locally and it's working.

**Comment by juliusdw** [comment]
ran it local, cant login but app starts.
Image is better.

**Review by juliusdw — APPROVED** [review]
ran it local, cant login but app starts.
Image is better.

**Pull request merged**